services/github/schemas/constants.yml

```diff
@@ -4,7 +4,7 @@
 # limitation, the identifiers won't be useful as routing keys in RabbitMQ
 # topic exchanges. Specifically, the length limitation and the fact that
 # identifiers can't contain dots `.` is critical.
```

Comment:
we should update this comment to tell that github service is doing sanitization and all dots are being replaced